react/ProductHighlights.tsx

```diff
@@ -122,6 +108,62 @@ const ProductHighlightContextProvider: FC<ProductHighlightContextProviderProps>
   )
 }
 
+ProductHighlights.propTypes = {
+  filter: PropTypes.object,
+  type: PropTypes.string
+}
+
+ProductHighlights.defaultProps = {
+  filter: {
+    type: 'hide',
+    highlightNames: [],
+  },
+  type: 'collection',
+}
+
+ProductHighlights.schema = {
+  title: 'Product Highlights',
+  type: 'object',
+  properties: {
+    filter: {
+      title: 'Filter',
+      type: 'object',
+      properties: {
+        type: {
+          title: 'type',
+          type: 'string',
+          enum: ['hide', 'show'],
+          enumNames: [
+            'hide',
+            'show'
```

Comment:
You should create messages for this values, since we support multiple languages.

Create a `messages` folder in the root of the project and add the `messages` in the `manifest.json` file. After this, add properties to at least one language to represent the values. The files have to follow [this structure](https://github.com/vtex-apps/store-components/tree/master/messages).

Example:

At file `en.json` inside the `messages` folder
`admin/editor.product-highlights.filter.type.hide: 'Hide'`

After you've done that, at the schema, you can change it to:
```suggestion
            'admin/editor.product-highlights.filter.type.hide',
            'admin/editor.product-highlights.filter.type.show'
```